### PR TITLE
[CI] Fix the sonar check  that parameters are overwritten

### PR DESCRIPTION
**Pull request by liugddx**
*(Please specify the module before the PR name: [core] ... or [flink] ...)*

### Purpose

*(What is the purpose of the change, or the associated issue)*

### Tests

*(List UT and IT cases to verify this change)*

### API and Format 

*(Does this change affect API or storage format)*

### Documentation

*(Does this change introduce a new feature)*

### CODE REVIEW

**Comment by liugddx** [comment]
[#https://github.com/sofastack/sofa-rpc/issues/15](https://github.com/sofastack/sofa-rpc/issues/15)

**Comment by liugddx** [comment]
This is a subtask of #836

File: pom.xml [hunk 1]
```diff
@@ -594,8 +595,6 @@ under the License.
                             <shadeTestJar>true</shadeTestJar>
                             <shadedArtifactAttached>false</shadedArtifactAttached>
                             <createDependencyReducedPom>true</createDependencyReducedPom>
-                            <dependencyReducedPomLocation>${project.basedir}/target/dependency-reduced-pom.xml
```
Comment (JingsongLi) [inline pom.xml:597]: Do we need a `maven-clean-plugin` like https://github.com/apache/flink/commit/60101e120fb2bd2393e9da00f07e1d53cec35c23

Comment (liugddx) [reply pom.xml:597]: > Do we need a `maven-clean-plugin` like [apache/flink@60101e1](https://github.com/apache/flink/commit/60101e120fb2bd2393e9da00f07e1d53cec35c23)

Done.

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**